docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -10,16 +10,62 @@ it does not cause running queries to fail; instead new queries become queued.
 
 The resource groups and associated selection rules are configured by a manager, which is pluggable.
-Add an ``etc/resource-groups.properties`` file with the following contents to enable
-the built-in manager that reads a JSON config file:
+
+You can use a file-based or a database-based resource group manager:
+
+* Add a file `etc/resource-groups.properties`
+* Set the `resource-groups.configuration-manager` property to `file` or `db`
+* Add further configuration properties for the desired manager.
+
+File resource group manager
+---------------------------
+
+The file resource group manager reads a JSON configuration file, specified with
+`resource-groups.config-file`:
 
 .. code-block:: text
 
     resource-groups.configuration-manager=file
     resource-groups.config-file=etc/resource-groups.json
 
-Change the value of ``resource-groups.config-file`` to point to a JSON config file,
-which can be an absolute path, or a path relative to the Trino data directory.
+The path to the JSON file can be an absolute path, or a path relative to the Trino
+data directory. The JSON file only needs to be present on the coordinator.
+
+Database resource group manager
+-------------------------------
+
+The database resource group manager loads the configuration from a relational database. The
+supported databases are MySQL, PostgreSQL, and Oracle.
+
+.. code-block:: text
+
+    resource-groups.configuration-manager=db
+    resource-groups.config-db-url=jdbc:mysql://localhost:3306/resource_groups
+    resource-groups.config-db-user=username
+    resource-groups.config-db-password=password
+
+If any of the tables required for the database resource group manager do not exist when Trino
+starts, they will be created automatically when Trino starts.
+
+Database resource group manager properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+================================================ =========================================================== ===============
+Property Name                                    Description                                                 Default
+================================================ =========================================================== ===============
+``resource-groups.config-db-url``                Database URL to load configuration from.                    ``none``
+
+``resource-groups.config-db-user``               Database user to connect with.                              ``none``
+
+``resource-groups.config-db-password``           Password for database user to connect with.                 ``none``
+
+``resource-groups.max-refresh-interval``         Time period for which the cluster will continue to accept   ``1h``
+                                                 queries after refresh failures cause configuration to
+                                                 become stale.
+
+``resource-groups.exact-match-selector-enabled`` Boolean flag to enable usage of a selector requiring        ``false``
```

Comment:
would be worth mentioning what the "non-exact" case means.

Reply:
I made a pass at explaining this. Tricky to write concisely.